### ISSUE TITLE
Cannot add another IMAP Mail Account to SOGo

### ISSUE THREAD

**Issue by vakilando** [mailcow/mailcow-dockerized#1002]
I'm using postfix/dovecot/SOGo for several years now an wanted to set up an new machine with mailcow (because I think it's really good, safe, ready to use, still customizable and it has a nice admin interface).
I installed mailcow twice because I was not able to add another IMAP Mail Account from the SOGo Webmail interface. Unfortunately it didn't help.

- I have "`SOGoMailAuxiliaryUserAccountsEnabled = YES;`" in "`/mailcow-dockerized/data/conf/sogo/sogo.conf`" (default setting).

- I can see the Link to add another IMAP Mail Account from the SOGo Webmail interface

- I can add an account and when I click on "save" a message appears that everything is fine and successfull

- But coming back to the Mailfolder the additional Mail account isn't there.....

I think I tried everything:

- I don't see any errors in the logs with "`docker-compose logs -f --tail=250`" or "`docker-compose logs -f --tail=250 sogo-mailcow`" when adding and saving the account.

- Even the sogo-tool doesn't work. The command "`docker-compose exec sogo-mailcow /bin/bash -c /usr/sbin/sogo-tool user-preferences set defaults user@dom.de AuxiliaryMailAccounts '{"AuxiliaryMailAccounts": [{"password": "******", "userName": "imapaccount", "identities": [{"email": "mail@addr.de", "signature": "test signature", "fullName": "Vor Nachname"}], "encryption": "ssl", "serverName": "themailserver.de", "name": "NameOfAccount"}]}'`" ends up in an error I can't resolve - but it could be the reason....?
The error is: "`sogo-tool[358:358] ERROR(+[GCSFolderManager defaultFolderManager]): default 'OCSFolderInfoURL' is not configured.`"

- I also tried the command "`docker-compose exec sogo-mailcow /bin/bash -c /usr/sbin/sogo-tool -v dump-defaults`"
It should show somthing like this (on a normal SOGo installation):
`OCSFolderInfoURL = "mysql://sogo:passwd@localhost:3306/sogo/sogo_folder_info";`
`OCSSessionsFolderURL = "mysql://sogo:passwd@localhost:3306/sogo/sogo_session_folder";`
But again the error "`sogo-tool[837:837] ERROR(+[GCSFolderManager defaultFolderManager]): default 'OCSFolderInfoURL' is not configured.`" shows up...
- I even setup an new machine without mailcow to check my settings an the sogo-tool syntax and it worked fine (but there I used apache instaed of nginx).

Can someone help me??

**Comment by andryyy** [comment]
Strange. Adding an aux user account via UI seems to be broken in this SOGo nightly. Can you please create a bug report for SOGo? :-)

Your command works, just do it like this: 

```
docker-compose exec sogo-mailcow /bin/bash
gosu sogo /bin/bash
/usr/sbin/sogo-tool user-preferences set defaults user@dom.de AuxiliaryMailAccounts '{"AuxiliaryMailAccounts": [{"password": "******", "userName": "imapaccount", "identities": [{"email": "mail@addr.de", "signature": "test signature", "fullName": "Vor Nachname"}], "encryption": "ssl", "serverName": "themailserver.de", "name": "NameOfAccount"}]}'
```

**Comment by vakilando** [comment]
You are great, it works!!!
And you are right (broken nightly), in my _new_ test installation I _can_ add an imap account with the _sogo-tool_, but adding it from the SOGo Webmail interface seems to be successfull but coming back to the webinterface the additional Mail account isn't there..... 
I didn't test the webinterface way.... On my older SOGo Installation it still works, so I will do no update there.
I'll make a bug report!
THANKS!

**Comment by andryyy** [comment]
You are right, I can create a new account, but when I renavigate to this menu, it is gone. Thanks for filing a report!

**Comment by vakilando** [comment]
You are great, it works!!!
And you are right (broken nightly), in my _new_ test installation I _can_ add an imap account with the _sogo-tool_, but adding it from the SOGo Webmail interface seems to be successfull but coming back to the webinterface the additional Mail account isn't there..... 
I didn't test the webinterface way.... On my older SOGo Installation it still works, so I will do no update there.
I'll make a bug report!
THANKS!

**Comment by vakilando** [comment]
Adding an additional user account via SOGo Web-UI seems to be broken in the actual SOGo nightly. 
I created a [bug report for SOGo](https://sogo.nu/bugs/view.php?id=4384).
Adding an additional user account via sogo-tool works as described above.

**Comment by vakilando** [comment]
The bug was fixed with nightly 3.2.11

**Comment by andryyy** [comment]
Nice! :-) Thanks for your help.
